### DISCUSSION TITLE
Is it possible? Https:// redirection to http:// on blocked sites.

### DISCUSSION THREAD

**Discussion by rgdi** [AdguardTeam/AdGuardHome#7464] [Q&A] [answered]
Hey, I was wondering If It would be possible to make a redirection to http:// all the requests blocked made with https//, so an HTTP, block page should work. (Without cert, errors, as it's http).  Thank you in advance for the help.

**Comment by alexsannikov** [comment, marked as answer]
No, AdGuard Home is a DNS proxy server, it doesn't make any redirection.
